web-registry/src/components/organisms/CreditBatches.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableHead from '@material-ui/core/TableHead';
+import axios from 'axios';
+import moment from 'moment';
+import cx from 'clsx';
+
+import {
+  StyledTableContainer,
+  StyledTableCell,
+  StyledTableRow,
+  StyledTableSortLabel,
+} from 'web-components/lib/components/table';
+import { getComparator, stableSort, Order } from 'web-components/lib/components/table/sort';
+import Section from 'web-components/lib/components/section';
+import { truncateWalletAddress } from '../../lib/wallet';
+
+interface BatchRowData {
+  start_date: string | Date;
+  end_date: string | Date;
+  issuer: string;
+  batch_denom: string;
+  class_id: string;
+  total_amount: number;
+  amount_cancelled: number;
+  project_location: string;
+}
+
+interface HeadCell {
+  id: keyof BatchRowData;
+  label: string;
+  numeric: boolean;
+  wrap?: boolean;
+}
+
+const headCells: HeadCell[] = [
+  { id: 'start_date', numeric: false, label: 'date created', wrap: true }, // TODO: this should be date_created, but that is not available yet
```

Comment:
can we just remove this for now as discussed?

Reply:
removed